x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/search_indices.tsx

```diff
@@ -30,6 +43,27 @@ export const baseBreadcrumbs = [
 ];
 
 export const SearchIndices: React.FC = () => {
+  const { loadSearchEngines, loadSearchIndices, onSearchEnginesLoad, onSearchIndicesLoad } =
+    useActions(SearchIndicesLogic);
+  const { searchIndices, searchEngines } = useValues(SearchIndicesLogic);
+
+  useEffect(() => {
+    loadSearchIndices();
```

Comment:
nitpick(non-blocking): I generally prefer using a single InitPageAction that triggers these data-load actions, rather than having two separate effects that trigger specific data fetches. It's more easily maintainable going forward, and makes re-use of and preventing unnecessary API calls easier.